tests/UserInfoTest.php

```diff
@@ -23,7 +23,7 @@ public function testItCanFetchUserInfo(): void
 
         $response = $client->usersList(['limit' => 2]);
 
-        /* @var UsersListGetResponse200 $response */
+        // @var UsersListGetResponse200 $response
```

Comment:
Wrong change, I guess this phpdoc should be moved above the $response assignment

Reply:
Good catch, I fixed it